namespace-provisioner/about.hbs.md

```diff
@@ -71,12 +68,12 @@ Controller mode
         # additional parameters about dev-ns1 added via label/annotations or GitOps
         - name: dev-ns2
         # additional parameters about dev-ns1 added via label/annotations or GitOps
-    ```
+```
 
 GitOps mode
 : Description of GitOps mode
 
-    - In the GitOps mode, the list of desired namespaces used by the `provisioner` application to create resources in, is maintained in a Git repository as a ytt data values file as shown [in this sample file](https://github.com/vmware-tanzu/application-accelerator-samples/blob/main/ns-provisioner-samples/gitops-install/desired-namespaces.yaml). This file provides a declarative way to indicate which namespaces should be populated with resources. For more information, see the options in [Customize Install](customize-installation.md).
+    - In the GitOps mode, the list of desired namespaces used by the `provisioner` application to create resources in, is maintained in a Git repository as a ytt data values file as shown [in this sample file](https://github.com/vmware-tanzu/application-accelerator-samples/blob/main/ns-provisioner-samples/gitops-install/desired-namespaces.yaml). This file provides a declarative way to indicate which namespaces should be populated with resources. For more information, see the `Options if using GitOps` section in [Customize Install](customize-installation.md).
 
 ## <a id ='nsp-controller'></a>Namespace Provisioner controller
 
```

Comment:
nit: Line 82 -> [Use a different label selector than default](customize-installation.md#label-selector).
`label-selector` is not an id in that file, should be updated to point to the desired header that does exist
(should be `con-label-selector`?)

Reply:
Updated the id